requirements.txt

```diff
@@ -1,10 +1,101 @@
-web3==5.0.2
```

Comment:
I did a `pip freeze > requirements.txt` here. It added all subdependencies.
Alternatively, we could just add the packages I added (celery, redis) to this file and reduce the number of packages tracked in it

Reply:
oh yeah, fuck this . just add celery and redis by hand. pip freeze is what got me into trouble with the last one when i went to change something...